db/deploy/tables/application_rfi_data.sql

```diff
@@ -0,0 +1,56 @@
+-- Deploy ccbc:tables/application_rfi_data to pg
+
+begin;
+
+create table ccbc_public.application_rfi_data(
+  rfi_data_id integer references ccbc_public.rfi_data(id),
+  application_id integer references ccbc_public.application(id),
+  primary key(rfi_data_id, application_id)
+);
+
+create index application_rfi_data_rfi_data_id_idx on ccbc_public.application_rfi_data(rfi_data_id);
+
+create index application_rfi_data_application_id_idx on ccbc_public.application_rfi_data(application_id);
+
+-- Enable row-level security
+alter table ccbc_public.application_rfi_data force row level security;
+alter table ccbc_public.application_rfi_data enable row level security;
+
+do
+$grant$
+begin
+perform ccbc_private.grant_permissions('select', 'application_rfi_data', 'ccbc_analyst');
+perform ccbc_private.grant_permissions('insert', 'application_rfi_data', 'ccbc_analyst');
+
+end
+$grant$;
+
+-- RLS: can only see and insert its own records in rfi_data
+do
+$policy$
+begin
+
+perform ccbc_private.upsert_policy('ccbc_analyst_insert_application_rfi_data', 'application_rfi_data', 'insert', 'ccbc_analyst',
+'application_id in (select id from ccbc_public.application)');
+
+perform ccbc_private.upsert_policy('ccbc_analyst_select_application_rfi_data', 'application_rfi_data', 'select', 'ccbc_analyst',
+'application_id in (select id from ccbc_public.application)');
+
+perform ccbc_private.upsert_policy('ccbc_analyst_select_rfi_data',
+  'rfi_data', 'select', 'ccbc_analyst',
```

Comment:
If we're going to have the schema dynamically created when viewing and submitting the form, this is missing `ccbc_auth_user` select RLS to be able to view the RFI

Reply:
good point. I'll ad `ccbc_auth_user` select RLS

Reply:
Fixed